pkg/apis/litmuschaos/v1alpha1/chaosresult_types.go

```diff
@@ -44,6 +44,15 @@ type ChaosResultStatus struct {
 	ExperimentStatus TestStatus `json:"experimentstatus"`
 	// ProbeStatus contains the status of the probe
 	ProbeStatus []ProbeStatus `json:"probeStatus,omitempty"`
+	// History contains cumulative values of verdicts
+	History HistoryDetails `json:"history,omitempty"`
+}
+
+// HistoryDetails contains cumulative values of verdicts
+type HistoryDetails struct {
```

Comment:
These are instance statuses of a specific result. 

Let's name them as passedRuns & failedRuns instead